tools/proofers/dp_proof.js

```diff
@@ -235,9 +235,14 @@ function submitForm(form) {
     setTimeout(() => {
         // disable after form submits (setTimeout) since disabled form values aren't sent and we
         // need the submit button to submit to determine which submit button was clicked
+        // standard interface buttons have type 'submit'
         form.querySelectorAll('input[type=submit]').forEach(inputSubmit => {
             inputSubmit.disabled = true;
         });
+        // enhanced interface buttons have type 'image'
+        form.querySelectorAll('input[type=image]').forEach(inputSubmit => {
```

Comment:
why do we need to disable images? do they have click handlers on them that submit the form?

Reply:
That was my understanding from looking at the enhanced interface core. Without that, double clicking the wordcheck image button  causes the error we're trying to avoid.

Reply:
> That was my understanding from looking at the enhanced interface code. Without that, double clicking the wordcheck image button causes the error we're trying to avoid.

I'm glad you were able to verify this -- I couldn't get the error in the enhanced interface at all on browserstack, Windows or maOS, even using the oldest supported browsers. (And good that you remembered -- I was just testing in the standard interface.

Reply:
Ah, checked it out, this is correct. input type=image behave like submit buttons per https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/image